src/server.py

```diff
@@ -1,26 +1,19 @@
-# _*_ coding: utf-8 _*_
+# _*_ Coding: utf-8 _*_
 
 import socket
-import io
-import pdb
+import sys
 import os.path
 import os
+import io
+import pdb
```

Comment:
This should *never* appear on a line by itself in your imports.  You really want to isolate importing and calling the `set_trace()` function to a single line:

```python
import pdb; pdb.set_trace()
```

Always do it that way.  Never import it at the top and then add the breakpoint later.  It's a recipe for madness trying to hunt down breakpoints before you check in your code if you do.